src/manifest/msquic.clog.cs

```diff
@@ -16,6 +16,11 @@ public class Types
     {
         public static string ADDR(byte[] value)
         {
+            if (value.Length == 0)
```

Comment:
Hmmm... Where do we log a NULL address? This same fix is likely needed for WPA parsing as well.

Reply:
https://github.com/microsoft/msquic/blob/main/src/platform/datapath_linux.c#L1601

Reply:
We do the same thing in Windows and Kernel too.